pkg/locking/control_plane_init_mutex.go

```diff
@@ -14,33 +14,34 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
+// Package locking implements locking functionality.
```

Comment:
This file is from kcp, and copied everywhere, e.g. k3s provider, etcdadm-bootstrap-provider, cluster-api-provider-rke2

Shall we raise an issue to CAPI, ask if it's feasible to set this as public?

Reply:
I will raise an issue